### ISSUE TITLE
Shell is not a LinearRing

### ISSUE THREAD

**Issue by Biaggio74** [Toblerity/Shapely#568]
## Expected behavior and actual behavior.

I would like to create a polygon with given coordinates, as per the User Manual. I am expecting to return a polygon  as shapely.geometry object
It works perfectly with other types of geometries, like points.

## Steps to reproduce the problem.

The following script was written in a python3 shell. 
`
Python 3.6.4 (v3.6.4:d48ecebad5, Dec 18 2017, 21:07:28) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from shapely.geometry import Polygon
>>> polygon = Polygon([(0, 0), (1, 1), (1, 0)])
Shell is not a LinearRing
>>> `

## Operating system

macOS High Sierra, version 10.13.3

## Shapely version and provenance

~ $ pip3 show shapely
Name: Shapely
Version: 1.6.4.post1
Summary: Geometric objects, predicates, and operations
Home-page: https://github.com/Toblerity/Shapely
Author: Sean Gillies
Author-email: sean.gillies@gmail.com
License: BSD
Location: /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages
Requires:

**Comment by sgillies** [comment]
I can't reproduce this bug report on my macbook.

Here's my program

```python
# test_shapely.py
from shapely.geometry import Polygon

polygon = Polygon([(0, 0), (1, 1), (1, 0)])
print(polygon)
```

And here's the result of running it with Python 3.6 in a new virtual environment. I'm using the binary wheel on PyPI as you can see below.

```
vas-y:tmp seang$ mkvirtualenv -p /usr/local/bin/python3.6 foo
Running virtualenv with interpreter /usr/local/bin/python3.6
Using base prefix '/Library/Frameworks/Python.framework/Versions/3.6'
New python executable in /Users/seang/envs/foo/bin/python3.6
Also creating executable in /Users/seang/envs/foo/bin/python
Installing setuptools, pip, wheel...done.
virtualenvwrapper.user_scripts creating /Users/seang/envs/foo/bin/predeactivate
virtualenvwrapper.user_scripts creating /Users/seang/envs/foo/bin/postdeactivate
virtualenvwrapper.user_scripts creating /Users/seang/envs/foo/bin/preactivate
virtualenvwrapper.user_scripts creating /Users/seang/envs/foo/bin/postactivate
virtualenvwrapper.user_scripts creating /Users/seang/envs/foo/bin/get_env_details
(foo) vas-y:tmp seang$ pip install shapely
Collecting shapely
  Downloading Shapely-1.6.4.post1-cp36-cp36m-macosx_10_9_intel.macosx_10_9_x86_64.whl (1.5MB)
    100% |████████████████████████████████| 1.6MB 72kB/s
Installing collected packages: shapely
Successfully installed shapely-1.6.4.post1
(foo) vas-y:tmp seang$ python test_shapely.py
POLYGON ((0 0, 1 1, 1 0, 0 0))
```

@Biaggio74 how did you install Shapely?

**Comment by Biaggio74** [comment]
I've installed shapely like this:
$ pip install shapely

I am getting closer now as I created a fresh python virtualenv. In the virtualenv I did everything the same and the code runs properly. Just like yours ;)
But I am also using Jupyter Lab for visualisation of my maps, through Matplotlib and there I am unable to run the same code. Even if I start the Jupyter notebook in the same environment with:
$ jupyter lab

**Comment by Biaggio74** [comment]
@perrygeo here is what I did now to make it work.

- uninstall shapely and rasterio from the global space
- create a fresh new virtual env
- install shapely, rasterio,  AND jupyterlab  in the virtual env
Now it works for me. It seems even if I started Jupyter lab from the virtual env,  it was running on the global version, because I did not install it before to the venv. (my mistake) But since it did not complained I was fine. :)

only one error came back which I had before:
when `src.transform * (0,0)` I am getting this error:
`---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-15-26209af71ddf> in <module>()
      5 print(src.index(39.1,-6.88)[0])
      6 print(src.index(39.1,-6.88)[1])
----> 7 print(src.transform * (0,0))

TypeError: can't multiply sequence by non-int of type 'tuple'`
